gpr/include/sim/gpr.hpp

```diff
@@ -4,20 +4,69 @@
 #include <array>
 #include <cassert>
 #include <cstdint>
+#include <type_traits>
 
 #include <sim/common.hpp>
 
 namespace sim {
 namespace gpr {
 
-inline constexpr std::size_t GPR_NUMBER = 32;
+inline constexpr size_t GPR_NUMBER = 32;
```

Comment:
Why left std::?

Reply:
I decided to use primitive types without std:: addition